app/views/questions/show.html.erb

```diff
@@ -24,7 +23,7 @@
   </div>
   <div class="pull-left">
   <% if @node.author && @node.author.photo_file_name %>
-    <img class="rounded-circle" src="<%= @node.author.photo_path(:thumb) %>" style="width:50px;margin-right:10px;" />
+    <img class="rounded-circle" src="<%= @node.author.photo_path(:thumb) %>" style="width:150px;margin-right:10px;" />
```

Comment:
Ah, we had actually reverted this back to 50, as it had been too big! Maybe you've mistaken the screenshot of how it WAS with the screenshot of how we want it to be?